src/Microsoft.ML.Recommender/MatrixFactorizationTrainer.cs

```diff
@@ -195,52 +210,55 @@ public sealed class Arguments
         /// <summary>
         /// The name of variable (i.e., Column in a <see cref="IDataView"/> type system) used be as matrix's column index.
         /// </summary>
-        public readonly string MatrixColumnIndexName;
+        internal readonly string MatrixColumnIndexName;
 
         /// <summary>
         /// The name of variable (i.e., column in a <see cref="IDataView"/> type system) used as matrix's row index.
         /// </summary>
-        public readonly string MatrixRowIndexName;
+        internal readonly string MatrixRowIndexName;
 
         /// <summary>
         /// The name variable (i.e., column in a <see cref="IDataView"/> type system) used as matrix's element value.
         /// </summary>
-        public readonly string LabelName;
+        internal readonly string LabelName;
 
         /// <summary>
         /// The <see cref="TrainerInfo"/> contains general parameters for this trainer.
         /// </summary>
         public override TrainerInfo Info { get; }
 
         /// <summary>
-        /// Legacy constructor initializing a new instance of <see cref="MatrixFactorizationTrainer"/> through the legacy
-        /// <see cref="Arguments"/> class.
+        /// Constructor initializing a new instance of <see cref="MatrixFactorizationTrainer"/> through the <see cref="Options"/> class.
```

Comment:
>Constructor initializing [](start = 12, length = 24)

replace with "Initializes". 

That seems to be standard for .net docs for ctors.